### DISCUSSION TITLE
Testing for signs of life that webhook jobs are executed

### DISCUSSION THREAD

**Discussion by hondaman900** [spatie/laravel-stripe-webhooks#138] [Q&A]
I have a similar condition mentioned here a few times, where everything looks fine, code is correct, Stripe CLI and Dashboard all report everything as functioning with 200 responses, yet my job handlers don't activate. I have read others' solutions and none are my issue (job calls are correctly formatted with underscores). 

Most of these issues conclude with short references references to tests, and I see a tests folder in the repository (though Composer didn't install them in my app), but I don't know how to use these tests, and the readme section on "Testing" simply says "Composer test", so I'm getting the impression that there's a lot more to this, requiring prior knowledge about how to test and validate the operational aspects of my code and this package, but I don't know what I need to know.

Can someone point me in the right direction to any information as to how to run these tests? I'm not looking to consume your time asking to be tutored, just pointed towards resources that can connect the dots for me, as this "everything seems to work but no results" situation has me dead in the water.

Thanks in advance.

**Comment by freekmurze** [comment]
We actually made an entire course on testing that you could check out: https://testing-laravel.com

There are comparable courses out there too if you don't like this one.

I can say that learning how to test will make you a much much better developer.  Here why I think learning how to test is very important: https://testing-laravel.com/importance-of-testing

**Reply by hondaman900** [reply]
Thank you, I'll check out both of those resources. Hopefully that will help me find the root of this issue.
